src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -0,0 +1,74 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <UsingTask TaskName="Microsoft.DotNet.PackageValidation.ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+  <UsingTask TaskName="Microsoft.DotNet.PackageValidation.GetLastStablePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
+
+  <PropertyGroup Condition="'$(EnablePackageBaselineValidation)' == 'true' and '$(IsCrossTargetingBuild)' == 'true'">
+    <PackageValidationIntermediateBaselineFile>$(BaseIntermediateOutputPath)PackageValidationBaselinePath.txt</PackageValidationIntermediateBaselineFile>
+    <!-- Static graph restore invokes CollectPackageDownloads from an outer build, non static graph restore doesn't.
+         To avoid multiple target invocations, hook onto an outer build extension point for non static graph restore. -->
+    <AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets Condition="'$(IsGraphBuild)' == 'true'">CollectPackageDownloads</AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets>
+    <AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets Condition="'$(IsGraphBuild)' != 'true'">_GenerateRestoreDependencies</AddPackageValidationBaselinePackageAsPackageDownloadBeforeTargets>
+  </PropertyGroup>
+
+  <Target Name="RunPackageValidation"
+          AfterTargets="Pack">
+    <PropertyGroup Condition="'$(EnablePackageBaselineValidation)' == 'true' and
+                              '$(PackageValidationBaselinePath)' == ''">
+      <PackageValidationBaselineVersion>$([System.IO.File]::ReadAllText('$(PackageValidationIntermediateBaselineFile)').Trim())</PackageValidationBaselineVersion>
+      <PackageValidationBaselinePath>$([MSBuild]::NormalizePath('$(NuGetPackageRoot)', '$(PackageValidationBaselineName)', '$(PackageValidationBaselineVersion)', '$(PackageValidationBaselineName).$(PackageValidationBaselineVersion).nupkg'))</PackageValidationBaselinePath>
+    </PropertyGroup>
+
+    <!-- PackageTargetPath isn't exposed by NuGet: https://github.com/NuGet/Home/issues/6671. Not using
+         PackageAbsoluteOutputPath to not depend on NuGet's internal _GetAbsoluteOutputPathsForPack target. -->
+    <Microsoft.DotNet.PackageValidation.ValidatePackage
+      PackageTargetPath="$([MSBuild]::ValueOrDefault('$(PackageTargetPath)', '$(PackageOutputPath)$(PackageId).$(PackageVersion).nupkg'))"
+      RuntimeGraph="$(RuntimeGraph)"
```

Comment:
Should we use?
https://github.com/dotnet/sdk/blob/c53c8ac81e1bcb8118c31bc4f5622396c72e9959/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props#L23